templates/acr/v1/README.md

```diff
@@ -6,25 +6,31 @@ Create or update a Container Registry.
 
 ## Parameters
 
-Parameter name | Description
--------------- | -----------
-registryName   | Required. The name of the container registry.
-location       | Optional. The location to deploy the container registry.
-registrySku    | Optional. The container registry SKU.
-tags           | Optional. Tags to apply to the resource.
+Parameter name | Required | Description
+-------------- | -------- | -----------
+registryName   | Required | Required. The name of the container registry.
+location       | Optional | Optional. The location to deploy the container registry.
+registrySku    | Optional | Optional. The container registry SKU.
+tags           | Optional | Optional. Tags to apply to the resource.
 
 ### registryName
 
+Required
```

Comment:
@vicperdana We could also add a link in like this. https://img.shields.io/badge/parameter-required-orange?style=flat-square

Reply:
Added the badges.